### ISSUE TITLE
Pagination Not working

### ISSUE THREAD

**Issue by sanyam1992000** [farazkhanfk7/E-Commerce-Django#10]
Pagination on Homepage of the deployed website is not working.

**Comment by farazkhanfk7** [comment]
Yes, I didn't added the pagination feature. Also there are not many items in the database of deployed website.
